### PR TITLE
Made print and warn work properly. Also fixed some intendation.

### PR DESCRIPTION
**Pull request by ccuser44**
The testservice messaging is currently disabled until I find a way to iterate over varargs.

### CODE REVIEW

**Comment by Sceleratis** [comment]
What is this fixing? They already work correctly.

**Comment by ccuser44** [comment]
- Reverted my change which was told by me to @fxeP1 

@Sceleratis They did not work properly at all. The original functions ignored nil arguments. Printed arguments as seperate messages and did not work with new output

**Comment by Sceleratis** [comment]
> * Reverted my change which was told by me to @fxeP1
> 
> @Sceleratis They did not work properly at all. The original functions ignored nil arguments. Printed arguments as seperate messages and did not work with new output

I think I did the multiple messages thing intentionally but it's been a while so idk

**Comment by ccuser44** [comment]
> I think I did the multiple messages thing intentionally but it's been a while so idk

Well it still changes the behavior of the functions in an undesirable way. also it doesnt eork with robloxes new studio output properly

File: MainModule/Server/Server.lua [hunk 1]
```diff
@@ -100,30 +100,30 @@ local ServiceVariables = {}
...
+--local message = function(...) local Str = "" game:GetService("TestService"):Message(Str) end
+local print = function(...) --[[if server.Core and server.Core.DebugMode then message("::DEBUG:: Adonis ::", ...) else]] print(":: Adonis ::", ...) --[[end]] end
+local warn = function(...) --[[if server.Core and server.Core.DebugMode then message("::DEBUG:: Adonis ::", ...) else]] warn(":: Adonis ::", ...) --[[end]] end
+local cPcall = function(func, ...) local function cour(...) coroutine.resume(coroutine.create(func),...) end local ran,error = pcall(cour,...) if error then warn(error) logError("SERVER",error) warn(error) end end
+local Pcall = function(func, ...) local ran,error = pcall(func,...) if error then warn(error) logError("SERVER",error) warn(error) end end
+local Routine = function(func, ...) coroutine.resume(coroutine.create(func),...) end
+local sortedPairs = function(t, f) local a = {} for n in next, t do table.insert(a, n) end table.sort(a, f) local i = 0 local iter = function () i = i + 1 if a[i] == nil then return nil else return a[i], t[a[i]] end end return iter end
+local GetEnv, GetEnv = function(env, repl)
```
[the remark this answers is not in the document]
Comment (ccuser44) [reply MainModule/Server/Server.lua]: Fixed

**Pull request merged**